src/gpgservice.py

```diff
@@ -0,0 +1,30 @@
+import gnupg  
+import json
+
+class GPGService:
+
+	def __init__(self):
+		self.gpg = gnupg.GPG(gnupghome = 'homegpg')
+		self.gpg.encoding = 'utf-8'
+
+	def generate_keys(self):
+		key = self.gpg.gen_key(self.gpg.gen_key_input(name_email = ''))
+		return key.fingerprint
+
+	def get_public_key(self, fingerprint):
+		return self.gpg.export_keys(fingerprint)
```

Comment:
What will be returned here?

1. will it be a list of keys or just a single key?
2. is it something that will be accepted directly by `add_conversation`?